src/scala/org/pantsbuild/scoverage/report/BUILD

```diff
@@ -0,0 +1,14 @@
+scala_library(
+  provides=scala_artifact(
+      org='org.pantsbuild',
+      name='scoverageReport',
```

Comment:
JVM artifacts tend to have dashed `name`s. So maybe `scoverage-report-generator`.